src/test/java/com/epam/definitions/ReflectionDefinitions.java

```diff
@@ -22,90 +17,49 @@
     private Object object;
 
     @Given("Class name - {string}")
-    public void intClass(String className) {
-
-        try {
-            clazz = Class.forName(className);
-            object = clazz.getConstructor().newInstance();
-        } catch (ClassNotFoundException | NoSuchMethodException | IllegalAccessException | InstantiationException | InvocationTargetException e) {
-            e.printStackTrace();
-        }
+    public void intClass(final String className) {
 
+        clazz = getClazz(className);
+        object = getNewInstance(clazz);
     }
 
     @Then("Verify if current Object.getName equals {string}")
-    public void verifyClassName(String className) {
+    public void verifyClassName(final String className) {
 
         assertEquals(clazz.getName(), className);
     }
 
     @And("Set value - {int} in private field - {string}")
-    public void setValueInPrivateField(int fieldValue, String fieldName) {
-
-        try {
-            Field field = clazz.getDeclaredField(fieldName);
-            boolean flag = field.isAccessible();
-
-            field.setAccessible(true);
-            field.set(object, fieldValue);
-            field.setAccessible(flag);
-        } catch (NoSuchFieldException | IllegalAccessException e) {
-            e.printStackTrace();
-        }
+    public void setValueInPrivateField(final int fieldValue, final String fieldName) {
 
+        setIntPrivateField(object, fieldName, fieldValue);
     }
 
     @Then("Verify if method - {string} return value - {int}")
-    public void verifyReturnValueOfMethod(String methodName, int fieldValue) {
+    public void verifyReturnValueOfMethod(final String methodName, final int fieldValue) {
```

Comment:
Rename fieldValue to expectedValue. 
Method return some abstract value, so no need to specify